docs/md/README.md

```diff
@@ -278,27 +280,19 @@ the services WFL talks to, and are named accordingly.
 
 #### Test code
 
-There are some unit tests under `./test/zero/`.
-
-| File                      | Test the namespace        |
-| ------------------------- | ------------------------- |
-| gcs<sub>test</sub>.clj    | zero.gcs in gcs.clj       |
-| pubsub<sub>test</sub>.clj | zero.pubsub in pubsub.clj |
-
-Run them with `boot test`.
-
-There are some integration tests under `integration`. These are not yet
-automated so we suggest running them locally before submitting.
-To run the integration tests, you'll need a `wfl` server running locally:
-
+Test code lives under the project `test/` root. At present, `wfl` has two kinds
+of test, `unit` and `integration`. These can be run via the `deps.edn`,
+optionally specifying the kind:
+ 
 ```shell
-./ops/server.sh
+clojure -A:test [unit|integration]
```

Comment:
Oh nice didn't realize we could rely just on the command line args, I will steal this setup for PTC :)